types/mui-datatables/index.d.ts

```diff
@@ -105,7 +105,7 @@ export interface MUIDataTableColumnOptions {
     hint?: string;
     customHeadRender?: (columnMeta: MUIDataTableCustomHeadRenderer, updateDirection: (params: any) => any) => string;
     customBodyRender?: (value: any, tableMeta: MUIDataTableMeta, updateValue: (s: any, c: any, p: any) => any) => string | React.ReactNode;
-    setCellProps?: (cellValue: string, rowIndex: number, columnIndex: number) => string;
+    setCellProps?: (cellValue: string, rowIndex: number, columnIndex: number) => object;
```

Comment:
[The documentation](https://github.com/gregnb/mui-datatables#column-options) says `setCellProps` returns a string. Can you back this with a code snippet from the library? I based my typings primarily on the documentation.

Reply:
I opened a PR to fix it in the documentation as well. It has been merged 10 hours ago :)

Reply:
I see it: https://github.com/gregnb/mui-datatables/pull/431. Approving review.